ui/src/app/zones/views/ZoneDetails/ZoneDetailsForm/ZoneDetailsForm.tsx

```diff
@@ -57,8 +74,8 @@ const ZoneForm = ({ id, closeForm }: Props): JSX.Element | null => {
           );
         }}
         resetOnSave={true}
-        saved={saved}
-        saving={saving}
+        saved={updated}
+        saving={updating}
```

Comment:
As above, you wouldn't ever want your component to end up in an inconsistent state (e.g.  both saved and saving at the same time).

Using a single status string/enum would help make illigal states such as this impossible and allow to simplify the codebase.

Reply:
Agreed.